app/views/posts/index.html.erb

```diff
@@ -11,12 +11,18 @@
             
 
 <ul>
+
+  <% if @user == current_user %>
+    <%= link_to 'Add a new post', new_user_post_path %>
```

Comment:
- [ ] I'm unable to find the button to add new post to the UI.. Here are a few screenshots of the application

![Screenshot from 2022-09-02 16-03-41](https://user-images.githubusercontent.com/46388113/188196534-75b75efc-b2bc-442d-aece-538d7c6cf8c3.png)

![Screenshot from 2022-09-02 15-59-51](https://user-images.githubusercontent.com/46388113/188196746-f2507fd9-eff8-4215-8f29-60a3a59ded3c.png)